src/views/Generator/TestOptions/LoadZones/LoadZones.tsx

```diff
@@ -59,6 +108,25 @@ export function LoadZones() {
           </RadixLink>
           .
         </Text>
+
+        <FieldGroup name="distribution" label="Distribution" errors={errors}>
```

Comment:
This is not a blocking comment, but IMO the test builder UI/copy could be improved. Some thoughts:
- I think the even/manual labels aren't clear enough
- Not sure how I feel about saving both the distribution type and the actual values when distribution is set to `even` - it almost looks like a purely UI setting 🤔. One idea I had was to make the fields editable, but have a "distribute evenly" button, but that would make the UX of adding new zones worse.